styles/jsTokens/atomic.tokens.js

```
@@ -1,209 +1,209 @@
+      "xl": "var(--rad-ui-grid-container-xl)"
     }
-};
+  },
+  "transition": {
+    "instant": "var(--rad-ui-transition-instant)",
+    "fast": "var(--rad-ui-transition-fast)",
+    "normal": "var(--rad-ui-transition-normal)",
+    "slow": "var(--rad-ui-transition-slow)",
+    "expressive": "var(--rad-ui-transition-expressive)"
+  }
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)